### ISSUE TITLE
Single point timeseries dataset marker is not shown.

### ISSUE THREAD

**Issue by renatodeleao** [apexcharts/apexcharts.js#235]
## Description
Single point `timeseries` chart displays an "apparently" empty chart.
Imagine a crowdfunding chart displaying pledge performance that has starting date A and end-date at B. On day one of the campaign (a), it may have a some point, only one entry (one pledge). That single point is not showing unless you hover the chart which displays the tooltip but not the `marker`.

## Solution
Maybe we could center the point in the middle of the chart, so it's displayed, like google analytics does. But i'm open to some more clever tricks

![sample image](https://trello-attachments.s3.amazonaws.com/5ba11f2f971d8e666cccc9f7/5c1105e2a7318b3695c8304c/4a6c90182aa65ec25874754adf940300/image.png)

## Alternatives
- Adding null values on the remaining days of this interval timeseries doens't solve.
- Since it's limited timeseries example from a to b, Adding 0 values to the remaining points will create the visual effect, but it's not 
- If you tweak the demo to use 2 points and two days, the `marker` dots for each point will appear (although cropped)

## Additional context
- Here's a [codepen](https://codepen.io/ohsimtabem/pen/JwRQXE) adapted from your timeseries demo illustrating the problem.

## Extra
- If this doesn't make sense or is the expected behaviour feel free to close it
- It looks like an edge case, but it happens in real-life scenearios
- thanks for your time developing this 🙌

**Comment by junedchhipa** [comment]
Good point (pun intended).
Area/Line charts usually require at least 2 data-points to connect and draw something. If there is just 1 data-point, area/line chart skips the drawing part.

So, to tackle the single point issue, we can conditionally convert the chart to scatter chart by checking if `data.length === 1`. Once, we have more than 1 data-point, we can switch to the area/line chart.

I have tried to mimic this behavior by tweaking some values. It's a very rough example, but it should give you a starting point to handle such scenarios.
Example - https://codepen.io/apexcharts/pen/REowPj

In the example, try adding one more item in the data array, and it will switch to the `area` chart.
Hope that helps!

**Comment by junedchhipa** [comment]
No worries at all. Glad the workaround helped :)

**Comment by arnvidhr** [comment]
Hello, this issue could happen. I sort of working with series, which by timeline not related with each other (eg. device data which read with very different time delays). And in some cases I have no any point at all (eg. I set single day, but specific device read only once per few days), sort of date filter set to too short time period, then I don't know how to draw just empty chart without any error, but could be good to have it.  For instance to draw empty chart, just then to load some data. Maybe I just don't know how.

Another thing about single point, as you write to move it to the center, what probably was already done. If i have several series, one of them with just single point, but other lines are good, with a lot of points, everything then getting pushed to the center and no healthy lines possible to see, only because of one line/serie, which only single point. 

BTW. I done Lithuanian localisation for this nice chart. Could put it somewhere in some branch maybe?
